misc/config_tools/schema/config.xsd

```diff
@@ -273,7 +273,8 @@ devices.</xs:documentation>
     </xs:element>
     <xs:element name="MAX_MSIX_TABLE_NUM" default="64">
         <xs:annotation>
-          <xs:documentation>Maximum number of MSI-X tables per device.</xs:documentation>
+          <xs:documentation>Maximum number of MSI-X tables per device,
+if this value is empty, then the default value would be calculated from board xml file.</xs:documentation>
```

Comment:
If this statement is correct, the ``default="64"`` must be removed from the line above.  (That's what's showing up in the generated documentation.)

Reply:
We still suggest the default value is 64, but user could choose to let config tool calculate based on board.xml.

Reply:
The doc process adds the value of ``default=value`` to the generated documentation. This will be confusing if the documentation has two default value statements for this option.  With the proposed change, the documentation will read:

- Maximum number of MSI-X tables per device, if this value is empty, then the default value
   would be calculated from board xml file. (Default value is 64)

Reply:
"Maximum number of MSI-X tables per device. If this value is empty, the default value **will** be calculated from the board XML file. (The default value is 64.)"

Reply:
```suggestion
If this value is empty, then the default value will be calculated from the board XML file.</xs:documentation>
```

Reply:
Updated.